e2e/windows/e2e-scenario.sh

```diff
@@ -159,14 +177,24 @@ jq --argjson JsonForVnet "$WINDOWS_VNET" \
     --arg ValueForCustomData "$CUSTOM_DATA" \
     --arg ValueForCSECmd "$CSE_CMD" \
     --arg ValueForVMSS "$DEPLOYMENT_VMSS_NAME" \
-    '.parameters += $JsonForVnet | .parameters += $JsonForLB | .parameters.galleries_AKSWindows_externalid.defaultValue=$ValueForImageExternalID | .resources[0] += $JsonForIdentity | .resources[0] += $JsonForSKU | .resources[0].properties.virtualMachineProfile.storageProfile+=$JsonForOSDisk | .resources[0].properties.virtualMachineProfile.networkProfile.networkInterfaceConfigurations[0] += $JsonForNetwork | .resources[0].properties.virtualMachineProfile.storageProfile.imageReference.id=$ValueForImageReference | .resources[0].properties.virtualMachineProfile.osProfile.adminPassword=$ValueForAdminPassword | .resources[0].properties.virtualMachineProfile.osProfile.customData=$ValueForCustomData | .resources[0].properties.virtualMachineProfile.extensionProfile.extensions[0].properties.settings.commandToExecute=$ValueForCSECmd | .parameters.virtualMachineScaleSets_akswin30_name.defaultValue=$ValueForVMSS' \
-    windows_vmss_template.json > $DEPLOYMENT_VMSS_NAME-deployment.json
+    '.parameters += $JsonForVnet | .parameters += $JsonForLB | 
```

Comment:
what is the change in this part? If it is only for format, please change it later in another PR.

Reply:
I have rolled back to the original version. For the scope of this PR reformatting is not necessary and hence reverted.

It is perhaps recommended to trim each line to have approximately 100 characters for better readability. (for example VS code has a plugin to do automatic trimming for C++)